### DISCUSSION TITLE
CORE parameter for C and CPP

### DISCUSSION THREAD

**Discussion by ArminJo** [SpenceKonde/ATTinyCore#543] [Development Plans]
Hi Spence, can you please add a line **build.versiondefines** like in https://github.com/SpenceKonde/megaTinyCore/blob/9beeed82b144f5b1fbd5d8384bf9a319cbdc10b8/megaavr/platform.txt#L16
also to this core?
[I have done it for my core also](https://github.com/ArminJo/DigistumpArduino/blob/a03c2fbba00023cb6761f0852f38e126341b85eb/digistump-avr/platform.txt#L16) 😀 .

Thanks
Armin

P.s. what is the semantics of` versionnum.released=1`?

**Comment by SpenceKonde** [comment]
I already have. See the combinethecores  branch, which is where what will
be 2.0.0 is being developed. Next version will be 2.0.0. I got distracted
by the newer parts though and did fresh releases of megaTinyCore and DxCore.

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***

**Comment by SpenceKonde** [comment]
Released is 0, unless that version is part of a release, in which case it
is 1. In theory, when a version is being made into a release, I change that
to 1, then when I bump version before making any code changes, I set it
back to 0. My record of tracking versions isnt very good, though.

The idea is to tell apart the code that I was working on while developing
2.3.0 and that was somewhere between 2.3.0 and 2.2.x from the final 2.3.0.in
any event, old versions of ATTinyCore shpuld still have the defines, just
pulled in Arduino.h and updated less consistently

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***
